staff/views/users.py

```diff
@@ -139,8 +139,12 @@ def form_valid(self, form):
         return redirect(self.user.get_staff_url())
 
     def get_form_kwargs(self):
+        existing_membership_orgs = self.user.orgs.values_list("pk", flat=True)
+        available_orgs = Org.objects.exclude(pk__in=existing_membership_orgs).order_by(
+            "name"
+        )
         return super().get_form_kwargs() | {
-            "available_orgs": Org.objects.order_by("name"),
+            "available_orgs": available_orgs,
```

Comment:
If we're only showing the organisations that a user is not a member of, how can we remove a user from an organisation?